packages/wp-source/src/actions.ts

```diff
@@ -1,20 +1,20 @@
 import WpSource from "../";
-import { parsePath } from "./utils/parse-path";
+import { routeToParams } from "./libraries/routeUtils";
 import { wpOrg, wpCom } from "./libraries/patterns";
 
 const actions: WpSource["actions"]["source"] = {
-  fetch: ({ state, libraries }) => async pathOrObj => {
+  fetch: ({ state, libraries }) => async routeOrParams => {
     const { source } = state;
     const { resolver } = libraries.source;
 
     let path: string;
     let page: number;
 
-    if (typeof pathOrObj === "string") {
-      ({ path, page } = parsePath(pathOrObj));
+    if (typeof routeOrParams === "string") {
```

Comment:
I would get rid of all this and I'd use only something like:
```js
const { path, page } = getParams(routeOrParams);
```

Reply:
Same here in router:
https://github.com/frontity/frontity/blob/dev/packages/tiny-router/src/actions.ts#L12-L17

```js 
const { path, page } = libraries.source.getParams(routeOrParams);
```